taskcluster/ci/ui-test/kind.yml

```diff
@@ -52,3 +52,42 @@ jobs:
                   path: /builds/worker/artifacts
                   type: directory
         worker-type: b-android
+    ui-test-screenshots-x86:
```

Comment:
@AaronMT @rpappalax so that this task does not run as a build task and so be a build blocker, should I create a new folder in taskcluster/ci/ui-test-screenshots/ and put there the kind.yml for this?

Reply:
Here is great! We put job that have about the same values in the same kind. We're in this case here, so let's keep this `ui-test-screenshots-x86` in this `kind.yml` 🙂

Reply:
Nit: Rename `ui-test-screenshots-x86` into `screenshots-x86`. I just realized the name of the task looks odd otherwise: https://firefox-ci-tc.services.mozilla.com/tasks/Rbj7yALeQy--ZhDx6kMmcw